google/oauth2/service_account.py

```diff
@@ -262,20 +273,28 @@ def requires_scopes(self):
         """
         return True if not self._scopes else False
 
-    @_helpers.copy_docstring(credentials.Scoped)
-    def with_scopes(self, scopes, default_scopes=None):
-        return self.__class__(
+    def _make_copy(self):
+        cred = self.__class__(
             self._signer,
             service_account_email=self._service_account_email,
-            scopes=scopes,
-            default_scopes=default_scopes,
+            scopes=self._scopes,
+            default_scopes=self._default_scopes,
```

Comment:
I think these objects need to be copied. Otherwise multiple objects will share pointers to these sequences

Reply:
yes coped them now